server/src/resolvers/args/ProjectArgs.ts

```diff
@@ -0,0 +1,119 @@
+import { Matches } from 'class-validator';
+import { ArgsType, Field } from 'type-graphql';
+import { ErrorMessages } from 'utils/enums/ErrorMessages';
+import { Validations } from 'utils/enums/Validations';
+
+const nameRegExp = new RegExp(Validations.USERNAME_REGEX);
+
+@ArgsType()
+export class getByIdArgs {
+	@Field()
+	projectId: string;
+}
+
+@ArgsType()
+export class getByNameArgs {
```

Comment:
add "project" or "projects" for all args classes too

Reply:
done